src/org/folio/rest_v2/Permissions.groovy

```diff
@@ -166,4 +166,29 @@ class Permissions extends Authorization {
         logger.info("Wait a minute for permissions cache update")
         sleep(60000)
     }
+
+/**
+ * Adds a missing permission to a user.
+ * source: RANCHER-895
+ * @param tenant The tenant in which the user is located.
+ * @param user The user to whom the permission is added.
+ * @param permissionName The permission to add.
+ */
+    void addUserMissingPermission(OkapiTenant tenant, OkapiUser user) {
+        def allPermissions = getAllPermissions(tenant)
```

Comment:
what you think to use this styling, more readable to know what type are returning from function
    List<String> allPermissions = getAllPermissions(tenant)
    List<String> existingPermissions = getUserPermissions(tenant, user)

Reply:
Hi Guram,
Void method doesn't return anything, except true or false, please refer to documentation of Groovy PL: https://groovy-lang.org/objectorientation.html
We don't need any return directive for this implementation.

Reply:
yes sorry , i different one mean
ignore 
closing conversation
thanks <3